src/utils/functions.js

```diff
@@ -78,6 +78,20 @@ const paramCountString = (params) => {
 	return result;
 };
 
+const renderProperties = (param) => (
+	param.properties ?
+		<div className={css.params}>
+			<h6><b>{param.name}</b>{` ➵ ${paramCountString(param.properties)}`}</h6>
```

Comment:
`h6` is already bold. Also, `<b>` is style, not semantic.
```
<h6>{param.name} ➵ {paramCountString(param.properties)}</h6>
```

That being said, I appreicate that you kept/continued the semantic definition list, term, and definition markup 👍

Reply:
Sounds good, I took this part out in response to Ryan's comment where it might not be useful and it clutters up the space.